cli/cmd.go

```diff
@@ -317,6 +317,7 @@ var Commands = []*cli.Command{
 	WithCategory("developer", logCmd),
 	WithCategory("developer", waitApiCmd),
 	WithCategory("developer", fetchParamCmd),
+	WithCategory("developer", disputerCmd),
```

Comment:
Probably shouldn't be a top-level command, maybe under chain -> `lotus chain diputer ...`